includes/core-changes/corefx/5.0/non-ascii-chars-in-uri-parsed-correctly.md

```diff
@@ -0,0 +1,54 @@
+### URI paths with non-ASCII characters parse correctly on Unix
+
+A bug was fixed in the <xref:System.Uri?displayProperty=fullName> class such that absolute URI paths that contain non-ASCII characters now parse correctly on Unix platforms.
+
+#### Change description
+
+In previous versions of .NET, absolute URI paths that contain non-ASCII characters are parsed incorrectly on Unix platforms, and segments of the path are duplicated. (Absolute paths are those that start with "/".) The parsing issue has been fixed for .NET 5.0. If you move from a previous version of .NET to .NET 5.0 or later, you'll get different values produced by <xref:System.Uri.AbsoluteUri?displayProperty=nameWithType>, <xref:System.Uri.ToString?displayProperty=nameWithType>, and other <xref:System.Uri> members.
+
+Consider the output of the following code when run on Unix.
+
+```csharp
+var myUri = new Uri("/üri");
+
+Console.WriteLine(myuri.AbsoluteUri);
```

Comment:
@gewarren I updated the casing with my suggestion and forgot to do it here and next line.